resources/assets/coffee/react/mp-history/event.coffee

```diff
@@ -51,4 +50,4 @@ class MPHistory.Event extends React.Component
               user.username
 
         span className: 'mp-history-event__text',
-          Lang.get "multiplayer.match.events.#{event_type}"
+          Lang.get "multiplayer.match.events.#{event_type}#{if !user? then '-no-user' else ''}"
```

Comment:
but wouldn't this [interpolate the user onto the string](https://puu.sh/qGIXy.png) when we don't want it to?